assets/collapsible-content.css

```diff
@@ -15,6 +15,10 @@
   }
 }
 
+.scroll-trigger .collapsible-content__media.gradient {
+  background: transparent;
+}
```

Comment:
I wonder if the background transparency should also be applied when the animation isn't on: 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/17-55-jp5mf-5w0p5.png)
</details>

Reply:
Yeah, probably. 👍

Reply:
Yeah, we should definitely apply transparency here. The full gradients are bound to look weird when they’re crammed into those short, long shapes: 

<img width="488" alt="Screenshot 2023-05-24 at 1 29 06 PM" src="https://github.com/Shopify/dawn/assets/1202812/81d80f46-b4ad-4209-8c2c-ef118c340d36">

...but also, [we display those accordions as transparent](https://screenshot.click/24-33-08jtv-fbosz.png) within the product details, so making them transparent here would allow merchants to have the Collapsible Content section match that.

Reply:
Oh sorry in my screenshots I meant for the image specifically and how the shadow either show or not behind the transparent image.

I think if the layout is set to row containers they shouldn't be set to transparent. Cause in this case you might as well set the layout to `No container`. I don't think it looks pretty, specifically with the radial gradient but that's up to the merchant to pick something nice. 
With a different gradient it doesn't look bad: 
![](https://screenshot.click/24-42-119ne-12p14.png)

Reply:
Change should be done for the image transparency 👍